### PR TITLE
Update dependencies version && router control && webpack HMR, devTools and CopyPlugin

### PR DESCRIPTION
**Pull request by wesleyalmd**

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+import { Provider } from 'react-redux';
+import store from '@/redux/store';
+import Routes from '@/routes';
+import App from '@/components/App';
+import './assets/css/styles.scss';
+
+console.log(Routes);
```
Comment (LucasCalazans) [inline src/index.jsx]: Please, remove this `console.log`

File: src/components/App.jsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-const App = props => (
-    <main className="app-main">
-        <header className="app-header">{ props.appName }</header>
-        <div className="app-container">
-            { props.children }
-        </div>
-    </main>
+const App = ({ appName, children }) => (
+  <main className="app-main">
+  {console.log(children)}
```
Comment (LucasCalazans) [inline src/components/App.jsx]: Please, remove this console.log

File: .editorconfig [hunk 3]
```diff
@@ -1,9 +0,0 @@
-# Editor configuration, see https://editorconfig.org
```
Comment (LucasCalazans) [inline .editorconfig]: Bring this file back, we need it

**Review by LucasCalazans — CHANGES_REQUESTED** [review]

**Comment by wesleyalmd** [comment]
@LucasCalazans, PR requires have been solved.

**Pull request closed without merging**